x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/index.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { ENTITY_TYPE } from '@kbn/observability-shared-plugin/common';
+import React from 'react';
+import type { Subject } from 'rxjs';
+import useEffectOnce from 'react-use/lib/useEffectOnce';
+import { InventoryGroupAccordion } from './inventory_group_accordion';
+import { useInventoryAbortableAsync } from '../../hooks/use_inventory_abortable_async';
+import { useKibana } from '../../hooks/use_kibana';
+import { InventorySummary } from './inventory_summary';
+import { useInventoryParams } from '../../hooks/use_inventory_params';
+import { useInventorySearchBarContext } from '../../context/inventory_search_bar_context_provider';
+
+export interface GroupedInventoryPageProps {
+  refresh: Subject<void>;
+}
+
+export function GroupedInventory() {
+  const {
+    services: { inventoryAPIClient },
+  } = useKibana();
+  const { query } = useInventoryParams('/');
+  const { kuery, entityTypes } = query;
+  const { refreshSubject$ } = useInventorySearchBarContext();
+
+  const { value = { groupBy: ENTITY_TYPE, groups: [] }, refresh } = useInventoryAbortableAsync(
+    ({ signal }) => {
+      return inventoryAPIClient.fetch('GET /internal/inventory/entities/group_by/{field}', {
+        params: {
+          path: {
+            field: ENTITY_TYPE,
+          },
+          query: {
+            kuery,
+            entityTypes: entityTypes?.length ? JSON.stringify(entityTypes) : undefined,
+          },
+        },
+        signal,
+      });
+    },
+    [entityTypes, inventoryAPIClient, kuery]
+  );
+
+  useEffectOnce(() => {
+    const refreshSubscription = refreshSubject$.subscribe(() => {
+      refresh();
+    });
+
+    return () => refreshSubscription.unsubscribe();
+  });
+
+  const totalEntities = value.groups.reduce((acc, group) => acc + group.count, 0);
```

Comment:
```suggestion
  const totalEntities = sumBy(value.groups, 'count')
```

Reply:
lodash utilities have significant overhead, both from payload size and also runtime performance. For a simple reduce/sum operation, it's a bit overkill.